sdk/src/main/java/io/dapr/utils/Retry.java

```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright 2021 The Dapr Authors
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package io.dapr.utils;
-
-class Retry {
-
-  private static final long RETRY_WAIT_MILLISECONDS = 1000;
-
-  private Retry() {
-  }
-
-  static void callWithRetry(Runnable function, long retryTimeoutMilliseconds) throws InterruptedException {
```

Comment:
this was used in tests and not being used anymore?

Reply:
It was only used in the NetworkUtils. So, I moved it there. I want to avoid ambiguity into people thinking this is the RetryPolicy.

Reply:
isn't this a breaking change? 
Though I believe that the users have to use the same package name, to access the method.

Reply:
Correct. Package here is used for internal scope.